qubes/vm/dispvm.py

```diff
@@ -41,6 +41,8 @@ class DispVM(qubes.vm.qubesvm.QubesVM):
                                 setter=_setter_template,
                                 doc='AppVM, on which this DispVM is based.')
 
+    template_for_dispvms = False
```

Comment:
I'd rather replace accessing the property with `getattr(vm, `template_for_dispvm`, False)`. In fact most of the places are this way already, there are just few that are not.

Reply:
What is the disadvantage of my approach?  Serious question.  Using `getattr` might cause difficulties if and when we add static types to our code.

Reply:
Nevermind, I forgot that `template_for_dispvms` is no ordinary Python attribute.